graphql_compiler/api/sql/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+# TODO: consider: import functions that help with setting up edges?
```

Comment:
Probably `from <file_in_this_directory> import *`, if someone really wants to actually import this at all?

Reply:
Not sure what this refers to - are you saying the imports shouldn't live here?